ports/yasm/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "yasm",
+  "version-semver": "1.3.0",
+  "port-version": 1,
+  "description": "Yasm is a complete rewrite of the NASM assembler under the “new” BSD License.",
```

```suggestion
  "description": "Yasm is a complete rewrite of the NASM assembler under the new BSD License.",
```

Let's get rid of the excessive unicode while we're here.